docs/api/app.md

```diff
@@ -1489,6 +1489,38 @@ This method can only be called after app is ready.
 
 Returns `Promise<string>` - Resolves with the proxy information for `url` that will be used when attempting to make requests using [Net](net.md) in the [utility process](../glossary.md#utility-process).
 
+### `app.setClientCertRequestPasswordHandler(handler)`  _Linux_
+
+* `handler` Function\<Promise\<string\>\>
+  * `clientCertRequestParams` Object
+    * `hostname` string - the hostname of the site requiring a client certificate
+    * `tokenName` string - the token (or slot) name of the cryptographic device
+    * `isRetry` boolean - whether there have been failed attempt at prompting the password
```

Comment:
```suggestion
    * `isRetry` boolean - whether there have been previous failed attempts at prompting the password
```

question: Would it be useful for users to have a Number count of how many previous retries have already happened instead of a boolean?

Reply:
I'm just exposing the same api as chromium is exposing to  implementors of the class. If the user wants to know the number of attempt, they can keep track of them. I know the same can be said for isRetry. But it's what chromium exposes.